src/App.tsx

```diff
@@ -1,24 +1,226 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useCallback,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
+import { getTodos, addTodo, removeTodo } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList/TodoList';
+import { TodoInput } from './components/TodoInput/TodoInput';
+import { FilterType } from './types/FilterType';
+import { Notification } from './components/Notification/Notification';
+import { ErrorType } from './types/ErrorType';
 
-const USER_ID = 0;
+const USER_ID = 7022;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterType, setFilterType] = useState<FilterType>(FilterType.All);
+  const [isError, setIsError] = useState<ErrorType>(ErrorType.None);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
+  const [loadingTodo, setLoadingTodo] = useState<number[]>([]);
+
+  const showError = (errorType : ErrorType) => {
+    setIsError(errorType);
+    setTimeout(() => setIsError(ErrorType.None), 3000);
+  };
+
+  const handleCloseError = useCallback(() => {
+    setIsError(ErrorType.None);
+  }, []);
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then(data => setTodos(data))
+      .catch(() => {
+        showError(ErrorType.Load);
+      });
+  }, []);
+
+  const activeTodos = useMemo(
+    () => todos.filter(todo => !todo.completed), [todos],
+  );
+
+  const completedTodos = useMemo(
+    () => todos.filter(todo => todo.completed), [todos],
+  );
+
+  const filteredTodos = useCallback(() => {
```

Comment:
better to use `useMemo` here. then you don;t have to call it like function
```suggestion
  const filteredTodos = useMemo(() => {
```